symbols.xml

```diff
@@ -1220,8 +1224,8 @@
         <global-address name='process_dig' value='0x141026a03'/>
         <global-address name='cursor' value='0x140e60058'/>
         <global-address name='selection_rect' value='0x140e60068'/>
-        <global-address name='gamemode' value='0x140e60088'/>
-        <global-address name='gametype' value='0x140e6008c'/> -4
+        <global-address name='gamemode' value='0x140e6008c'/>
+        <global-address name='gametype' value='0x140e60088'/>
```

Comment:
With my offsets I get the following on my Win64 build.
[lua]# ! df.global.gamemode
3
[lua]# ! df.global.gametype
11

The file had duplicates rows to start with. I'm not familiar enough with the scripts to know if that would cause issues.

Unrelated but 2 issues I have on the Win32 build that I dont have on the Win64 build with these pointers are:
1. The DFHack and Git text is truncated to 1 character.
2. The df generated seem to only be of 1 character width and shoved on left side. 

I assume these are same problem.  I can figure out how to join the irc channel if its better to discuss this there.  Not sure this is related to my offsets.